src/main/plugins/org.dita.pdf2/xsl/fo/links.xsl

```diff
@@ -407,9 +406,7 @@ See the accompanying license.txt file for applicable licenses.
           <fo:list-item xsl:use-attribute-sets="related-links.ul.li">
             <fo:list-item-label xsl:use-attribute-sets="related-links.ul.li__label">
               <fo:block xsl:use-attribute-sets="related-links.ul.li__label__content">
-                <fo:inline>
```

Comment:
I believe in many cases there is not a parent element. I initially noticed this on `<xref>` and `<link>` where there is no parent block, but the `<fo:inline>` is generated before the `<fo:basic-link>` container. I don't know if there is a reason to do it that way - if maybe at some point `@id` on `<fo:basic-link>` wouldn't create a valid link?